libs/ext2_fs/src/layout.rs

```diff
@@ -0,0 +1,64 @@
+use crate::tools::*;
+use serde::{Serialize, Deserialize};
+use serde_big_array::big_array;
+big_array! {
+    BigArray;
+    64, 190
+}
+
+/// Structure of the super block
+/// disk layout
+#[derive(Serialize, Deserialize, Debug, Eq, PartialEq)]
```

Comment:
Why did you end up with serde?

Reply:
This is the serialization tool used in another rust project I am studying, and after searching, I found that the article about this tool is relatively rich.

Reply:
I'm not sure if we can use serde for our usage as well. How do you plan to deal with endianess?